# HG changeset patch
# User Rik <rik@octave.org>
# Date 1352155854 28800
#      Mon Nov 05 14:50:54 2012 -0800
# Node ID 80a4ded6b89c18dd83451969802ca253c655d7f1
# Parent  ac86b891e45a6859cfdb044e54bac6f180f1b95f
warning_ids.m: Add 2 missing warning IDs to documentation.

warning_ids.m: Add remove-init-dir and deprecated-keyword warnings to
documentation.

diff --git a/scripts/miscellaneous/warning_ids.m b/scripts/miscellaneous/warning_ids.m
--- a/scripts/miscellaneous/warning_ids.m
+++ b/scripts/miscellaneous/warning_ids.m
@@ -126,16 +126,22 @@
 ## Warn when performing broadcasting operations.  By default, this is
 ## enabled.  See @ref{Broadcasting} in the chapter Vectorization and Faster
 ## Code Execution of the manual.
 ##
 ## @item Octave:built-in-variable-assignment
 ## By default, the @code{Octave:built-in-variable-assignment} warning is
 ## enabled.
 ##
+## @item Octave:deprecated-keyword
+## If the @code{Octave:deprecated-keyword} warning is enabled, a
+## warning is issued when Octave encounters a keyword that is obsolete and
+## scheduled for removal from Octave.
+## By default, the @code{Octave:deprecated-keyword} warning is enabled.
+##
 ## @item Octave:divide-by-zero
 ## If the @code{Octave:divide-by-zero} warning is enabled, a
 ## warning is issued when Octave encounters a division by zero.
 ## By default, the @code{Octave:divide-by-zero} warning is enabled.
 ##
 ## @item Octave:fopen-file-in-path
 ## By default, the @code{Octave:fopen-file-in-path} warning is enabled.
 ##
@@ -235,31 +241,40 @@
 ## do_braindead_shortcircuit_evaluation (1)
 ## @end group
 ## @end example
 ##
 ## @noindent
 ## if you would like to enable this short-circuit evaluation in
 ## Octave.  Note that the @code{&&} and @code{||} operators always short
 ## circuit in both Octave and @sc{matlab}, so it's only necessary to
-## enable @sc{matlab}-style short-circuiting it's too arduous to modify
+## enable @sc{matlab}-style short-circuiting if it's too arduous to modify
 ## existing code that relies on this behavior.
 ## By default, the @code{Octave:possible-matlab-short-circuit-operator} warning
 ## is enabled.
 ##
 ## @item Octave:precedence-change
 ## If the @code{Octave:precedence-change} warning is enabled, Octave
 ## will warn about possible changes in the meaning of some code due to
 ## changes in precedence for some operators.  Precedence changes have
 ## typically been made for @sc{matlab} compatibility.
 ## By default, the @code{Octave:precedence-change} warning is enabled.
 ##
 ## @item Octave:recursive-path-search
 ## By default, the @code{Octave:recursive-path-search} warning is enabled.
 ##
+## @item Octave:remove-init-dir
+## The @code{path} function changes the search path that Octave uses
+## to find functions.  It is possible to set the path to a value which
+## excludes Octave's own built-in functions.  If the
+## @code{Octave:remove-init-dir} warning is enabled then Octave will warn
+## when the @code{path} function has been used in a way that may render
+## Octave unworkable.
+## By default, the @code{Octave:remove-init-dir} warning is enabled.
+##
 ## @item Octave:reload-forces-clear
 ## If several functions have been loaded from the same file, Octave must
 ## clear all the functions before any one of them can be reloaded.  If
 ## the @code{Octave:reload-forces-clear} warning is enabled, Octave will
 ## warn you when this happens, and print a list of the additional
 ## functions that it is forced to clear.
 ## By default, the @code{Octave:reload-forces-clear} warning is enabled.
 ##
